pkg/codegen/docs/gen.go

```diff
@@ -816,12 +817,14 @@ func (mod *modContext) getProperties(properties []*schema.Property, lang string,
 		}
 
 		docProperties = append(docProperties, property{
+			ID:                 strings.ToLower(propLangName),
```

Comment:
It's possible the same property name is going to show up on the same page (e.g. on the resource itself and on one of the supporting types). Should we prefix the type name to help ensure the id is unique?

Reply:
yea, I think that may be a good idea.